hitrace/src/lib.rs

```diff
@@ -70,6 +70,23 @@ pub fn finish_trace() {
     finish_trace_()
 }
 
+pub fn count_trace<T: AsRef<CStr>>(name: &T, count: i64) {
```

Comment:
I'm wondering if we can choose a better name than the original. 
Maybe `record_metric`, `trace_metric`, `record_value`. 

Perhaps it would also be nice for users to make count be `Into<i64>` so that we can accept all integers that are guaranteed to fit into an `i64`.
And for the remaining integer types (u64, i128, u128, usize, isize), perhaps we could provide a `saturating` version of this function, that does a saturating conversion to `i64`.